spec/src/main/asciidoc/cdi.asciidoc

```diff
@@ -74,7 +74,7 @@ public interface MyServiceClient {
 
 The values of the following properties will be provided via MicroProfile Config:
 
-- `com.mycompany.remoteServices.MyServiceClient/mp-rest/url`: The base URL to use for this service, the equivalent of the `baseUrl` method.  This property is considered required, however implementations may have other ways to define these URLs.
+- `com.mycompany.remoteServices.MyServiceClient/mp-rest/url`: The base URL to use for this service, the equivalent of the `baseUri` method.  This property is considered required, however implementations may have other ways to define these URLs.
```

Comment:
should we also support `*/mp-rest/uri`?

Reply:
Good question... we probably should, and then deprecate the `*/mp-rest/url` config property (assuming the consensus is to deprecate the `baseUrl` method).  Any strong opinions?